src/core/Akka.Remote.Tests.MultiNode/RemoteQuarantinePiercingSpec.cs

```diff
@@ -0,0 +1,139 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="RemoteQuarantinePiercingSpec.cs" company="Akka.NET Project">
+//     Copyright (C) 2009-2016 Lightbend Inc. <http://www.lightbend.com>
+//     Copyright (C) 2013-2016 Akka.NET project <https://github.com/akkadotnet/akka.net>
+// </copyright>
+//-----------------------------------------------------------------------
+
+
+using System;
+using Akka.Actor;
+using Akka.Configuration;
+using Akka.Remote.TestKit;
+using Akka.TestKit;
+
+namespace Akka.Remote.Tests.MultiNode
+{
+    public class RemoteQuarantinePiercingConfig : MultiNodeConfig
+    {
+        public RoleName First { get; }
+        public RoleName Second { get; }
+
+        public RemoteQuarantinePiercingConfig()
+        {
+            First = Role("first");
+            Second = Role("second");
+
+            CommonConfig = DebugConfig(false)
+                .WithFallback(ConfigurationFactory.ParseString(@"
+                  akka.loglevel = INFO
+                  akka.remote.log-remote-lifecycle-events = INFO
+                "));
```

Comment:
This does not do what you think it does. 
First the correct values for log-remote-lifecycle-events is either 'on' or 'off'. 
See: https://github.com/akkadotnet/akka.net/blob/master/src/core/Akka.Remote/Configuration/Remote.conf

Second. The WithFallback does exactly that. Fallback. It does _not_ override any existing values. Which means that if the DebugConfig(false) loads from base, it will include all the base values from pigeon.conf. 
This fallback definition is effectively doing nothing. 

And only works because the defaults are exactly the same. Also it does not crash on akka.remote.log-remote-lifecycle-events because the fallback value is never evaluated, since the default is loaded.